.github/workflows/Specs.yml

```diff
@@ -0,0 +1,48 @@
+name: Specs
+
+jobs:
+  specs:
+    strategy:
+      matrix:
+        os: [ubuntu-16.04]
+        ruby: [2.3.3, 2.4, 2.5, 2.6, 2.7]
+        include:
+          - os: macos-10.15
+            ruby: system
+
+    name: Ruby ${{ matrix.ruby }}
```

Comment:
Can you include the OS in the workflow name? Makes it much easier to skim

Reply:
@endocrimes - I can't include just the OS type, and I figured that saying `ubuntu-16.04` is a bit verbose. Do you think it's better to include it, even if verbose?

Reply:
@endocrimes I have added the runner name to the job title - looks a bit verbose (e.g. `Specs / ubuntu-16.04 / Ruby 2.6 (pull_request)`) but livable.

Reply:
Verbose is good for things like this I think 😅  - Thanks for adding it!